js/modules/loki_message_api.js

```diff
@@ -118,15 +118,22 @@ class LokiMessageAPI {
     };
 
     const doRequest = async nodeUrl => {
-      const url = `${nodeUrl}${this.messageServerPort}/store`;
+      const url = `${nodeUrl}${this.messageServerPort}/v1/storage_rpc`;
+      const body = {
+        method: 'store',
+        params: {
+          pubKey,
+          ttl: ttl.toString(),
+          nonce,
+          timestamp: timestamp.toString(),
+          data: data64,
+        },
+      };
       const fetchOptions = {
         method: 'POST',
-        body: data64,
+        body: JSON.stringify(body),
```

Comment:
Is there a reason you made this change? from what i recall, it should automatically be converted to json later down the line (in nodeFetch)

Reply:
I was receiving `"[object object]" ` on the server side...
I also believed it would be converted automatically.
Also from the package examples, they seem to be explicitly convert manually:
https://www.npmjs.com/package/node-fetch#post-with-json

Reply:
Ah, makes sense 👍